wgs/community-membership.md

```diff
@@ -96,8 +96,8 @@ The following apply to the part of codebase for which one would be a reviewer in
 an [OWNERS] file (for repos using the bot).
 
 - member for at least 3 months
-- Primary reviewer for at least 5 PRs to the codebase
-- Reviewed or merged at least 20 substantial PRs to the codebase
+- Primary reviewer for at least 3 PRs to the codebase
```

Comment:
3 **substantial** PRs?

Reply:
@terrytangyuan do you have an alternative ?  I would like to update and finish this update.

Reply:
ok, I changed in "Reviewer for or author of at least 5 substantial PRs to the codebase" in 2nd commit